parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1033,10 +1033,12 @@ link = InputPattern
     ]
   )
   (\case
-    dest : srcs -> first fromString $ do
-      srcs <- traverse Path.parseHQSplit' srcs
-      dest <- Path.parseHQSplit' dest
-      Right $ Input.LinkI srcs dest
+    md : defs -> first fromString $ do
```

Comment:
Better nomenclature here, moving away from "source" and "target" to "metadata" and "definitions", as in, you "link/associate this metadata with these definitions".